docs/install.md

```diff
@@ -477,6 +479,32 @@ the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment via
 
 For beta versions of Tekton CRDs, setting `enable-api-fields` to "beta" is the same as setting it to "stable".
 
+## Enabling larger results using sidecar logs
+
+**Note**: The maximum size of a Task's results is limited by the container termination message feature of Kubernetes, as results are passed back to the controller via this mechanism. At present, the limit is per task is “4096 bytes”. All results produced by the task share this upper limit.
+
+To exceed this limit of 4096 bytes, you can enable larger results using sidecar logs. By enabling this feature, you will have a configurable limit (with a default of 4096 bytes) per result with no restriction on the number of results. The results are still stored in the taskrun crd so they should not exceed the 1.5MB CRD size limit.
+
+**Note**: to enable this feature, you need to grant `get` access to all `pods/log` to the `Tekton pipeline controller`. This means that the tekton pipeline controller has the ability to access the pod logs.
+
+1. Create a cluster role and rolebinding by applying the following spec to provide log access to `tekton-pipelines-controller`. 
+
+```
+kubectl apply -f config/enable-log-access-to-controller/
+```
```

Comment:
The Tekton release is produced doing:

```shell
ko resolve -R -f config/
```

We process all subfolders of `config` recursively. 
In fact, you can check the [latest nightly](https://storage.googleapis.com/tekton-releases-nightly/pipeline/latest/release.yaml) and you'll notice that the new `Role` and `RoleBinding` are already there and they would be installed by the standard Tekton installation.

This new `Role` to access logs was meant to be an opt-in, but it's installed by default right now.
We'll need to move those files to some other folder and also we will need to CI machinery to install them for testing this feature in CI on the alpha job.

Reply:
Yes, this new role was meant to be opt-in. I'm happy to move it to another directory. Thanks for the catch. I was following how the remote resolvers were set up.

Reply:
The remote resolvers are installed by default :)

Reply:
aha! Makes sense then. I will move it to a parallel folder `optional_config`?